security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -21,7 +21,7 @@ The following advisory is available for the {cert-manager-operator} 1.11.4:
 
 * link:https://access.redhat.com/errata/RHEA-2023:4081[RHEA-2023:4081]
 
-For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/[cert-manager project release notes for v1.11]. The golang version is updated to the version `1.19.10` to fix Common Vulnerabilities and Exposures (CVEs).
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/[cert-manager project release notes for v1.11]. The golang version is updated to the version `1.19.10` to fix Common Vulnerabilities and Exposures (CVEs). The `1.11.4` version of the {cert-manager-operator} is based on upstream cert-manager `v1.11.4`.
```

Comment:
This PR mentions wording upstream 1.11.1 and gives its link. However this PR mentions similar wording upstream 1.11.4 but aside of the 1.11.1's link directed by this PR, https://cert-manager.io/docs/release-notes/release-notes-1.11 has no 1.11.4 section at all. So line 24 confuses reader.
Only found https://github.com/cert-manager/cert-manager/releases/tag/v1.11.4 . Let's include it?

Reply:
I'll raise a PR upstream to https://github.com/cert-manager/website, to fix content in https://cert-manager.io/docs/release-notes/release-notes-1.11/ to include 1.11.4 information. This'd help remove ambiguity amongst users when seeking 1.11.1+ release notes.

Reply:
@xingxingxia @xenolinux FYR: https://github.com/cert-manager/website/pull/1268, once ti's merged https://cert-manager.io/docs/release-notes/release-notes-1.11/ should include 1.11.4 info.

Reply:
that PR got merged, so @xenolinux you can directly use this link: https://cert-manager.io/docs/release-notes/release-notes-1.11/ (it has 1.11.4 contents now :)
Hoping it helps remove the ambiguity!

Reply:
Thanks! @xingxingxia @swghosh I updated the PR with the v1.11.4 upstream link